core/gui/src/app/dashboard/component/user/list-item/list-item.component.ts

```diff
@@ -249,4 +250,6 @@ export class ListItemComponent implements OnInit, OnChanges {
       }
     }
   }
+
+  formatSize = formatSize;
```

Comment:
what is this for?

Reply:
I need to use the `formatSize` function from `size-formatter.util.ts` within the HTML file.

Reply:
Oh I see. Then can you add short comments to explain this?